src/htmlContent/video-view.html

```diff
@@ -0,0 +1,22 @@
+<div class="video-view">
+    <div class="video-data"></div>
+    <div class="video-wrapper">
+        <video controls>
+          <source src="{{ videoUrl }}" type="video/mp4">
+        </video>
+    </div>
+    <p>Embed Code</p>
+<pre>
```

Comment:
Yeah fair enough, I was thinking more about the experience for people viewing the published project.